scenarios/customapp_pod.yaml

```diff
@@ -0,0 +1,32 @@
+config:
+  runStrategy:
+    runs: 1
+    maxSecondsBetweenRuns: 30
+    minSecondsBetweenRuns: 1
+scenarios:
+  - name: "delete acme-air pods"
+    steps:
+    - podAction:
+        matches:
+          - labels:
+              namespace: "acme-air"
+              selector: ""
+        filters:
+          - randomSample:
+              size: 1
+        actions:
+          - kill:
+              probability: 1
+              force: true
+    - podAction:
+        matches:
+          - labels:
+              namespace: "acme-air"
+              selector: ""
+        retries:
+          retriesTimeout:
+            timeout: 180
+
+        actions:
+          - checkPodCount:
```

Comment:
@juliemathew we can skip the "scenarios/post_action_customapp_pod.py" action in favor of this check I guess. Thoughts?

Reply:
Yes. I agree. 
Made the changes. Please review. Thanks.